x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -2509,11 +2509,6 @@
     "kbnVislibVisTypes.pie.pieTitle": "パイ",
     "kbnVislibVisTypes.pie.segmentTitle": "スライスの分割",
     "kbnVislibVisTypes.pie.splitTitle": "チャートを分割",
-    "markdownVis.function.help": "Markdown ビジュアライゼーション",
-    "markdownVis.markdownDescription": "マークダウン構文でドキュメントを作成します",
-    "markdownVis.params.fontSizeLabel": "フォントサイズ ({fontSize} pt)",
-    "markdownVis.params.helpLinkLabel": "ヘルプ",
-    "markdownVis.params.openLinksLabel": "新規タブでリンクを開く",
```

Comment:
All these translations: should they be removed? or renamed? E.g:

```
"markdownVisType.params.openLinksLabel": "新規タブでリンクを開く",
```

Reply:
removed, then our translation team will add them back, the tool they use will find this as renamed.